### PR TITLE
Update prompt message to install integration library for worker

### PR DESCRIPTION
**Pull request by discdiver**
Current message makes it sound like a worker is being installed, not a library.

A code blurb is best. Changes to features should include an example that is executable by a new user.
If changing documentation, a link to a preview of the page is great.
 -->

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [X] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: tests/cli/test_worker.py [hunk 1]
```diff
@@ -503,10 +503,9 @@ async def test_install_policy_prompt(self, kubernetes_work_pool, monkeypatch):
             ],
             user_input=readchar.key.ENTER,
             expected_output_contains=[
-                (
-                    "Could not find a kubernetes worker in the current"
-                    " environment. Install it now?"
-                ),
+                "Could not find the Prefect integration library for the",
+                "kubernetes",
+                "Install the library now?",
```
Comment (zzstoatzz) [inline tests/cli/test_worker.py:508]: The original code here I think is what we want. your change is now checking for 3 distinct substrings of the output, which seems unnecessary

File: src/prefect/cli/worker.py [hunk 2]
```diff
@@ -316,8 +316,9 @@ async def _get_worker_class(
             # Confirm with the user for installation in an interactive session
             elif install_policy == InstallPolicy.PROMPT and is_interactive():
                 message = (
-                    f"Could not find a {worker_type} worker in the current"
-                    " environment. Install it now?"
+                    "Could not find the Prefect integration library for the"
+                    " {worker_type} worker in the current environment."
```
Comment (zzstoatzz) [inline src/prefect/cli/worker.py:320]: This needs to be an f-string, or else the output will literally be `{worker_type}`
```suggestion
                    f" {worker_type} worker in the current environment."
```

**Review by zzstoatzz — CHANGES_REQUESTED** [review]

File: tests/cli/test_worker.py (hunk 1 above)
Comment (discdiver) [reply tests/cli/test_worker.py:508]: Lol on n f string. The wrapping keeps getting screwed up, even copy pasting. Talked to Alex and decided to just make it separate list of strings.

Comment (zzstoatzz) [reply tests/cli/test_worker.py:508]: you could always just omit the commas

```python
In [1]: [
   ...:     "foo "
   ...:     "bar "
   ...:     "baz ",
   ...:     42
   ...: ]
Out[1]: ['foo bar baz ', 42]
```

**Review by zzstoatzz — CHANGES_REQUESTED** [review]
undoing my approval because I approved before the new changes were pushed here

**Comment by desertaxle** [comment]
@discdiver I think you want to merge into `main` instead of `flow-deploy`.

**Comment by discdiver** [comment]
Closing. Will re-open on main.

**Pull request closed without merging**